### PR TITLE
Multi language ui component

### LINKED ISSUE #190: Multi-language UI components
Visual Designs:
- https://app.zeplin.io/project/5878c3481bb7631c20cb161a/screen/5b05b3fce25151ce13d86d48
- https://app.zeplin.io/project/5878c3481bb7631c20cb161a/screen/5d63b15b38345e9b5a6d7de0

@DominikGuzei will advise on where to add the components, but this is not a blocker to begin development

### PR DESCRIPTION
**Pull request by DeeJayElly**
Multi language ui component
Closes #190 

screencapture-localhost-4000-2020-02-24-14_42_57
Screenshot 2020-02-24 15 59 01
Screenshot 2020-02-24 14 43 20

### CODE REVIEW

**Comment by a-rukin** [comment]
@DeeJayElly I've updated language switch design -> border width and some small padding tweak, as it was an old element and was broken a bit during the "copy-paste" phase. Pls take a look

**Comment by DeeJayElly** [comment]
> @DeeJayElly I've updated language switch design -> border width and some small padding tweak, as it was an old element and was broken a bit during the "copy-paste" phase. Pls take a look

OK i will.

Thank you for letting me know

**Comment by DeeJayElly** [comment]
> The Component and it's variations should appear in Storybook without the header

Done @rhyslbw

**Comment by DeeJayElly** [comment]
> @DeeJayElly I've updated language switch design -> border width and some small padding tweak, as it was an old element and was broken a bit during the "copy-paste" phase. Pls take a look

Done

**Comment by a-rukin** [comment]
image
triangle is too shifted to the bottom

**Comment by a-rukin** [comment]
@DeeJayElly also explained here https://zpl.io/V4MNrdQ the opened dropdown view 
p.s. this should be a quick fix

**Comment by DeeJayElly** [comment]
> image
> 
> triangle is too shifted to the bottom

Fixed

**Comment by DeeJayElly** [comment]
> @DeeJayElly also explained here https://zpl.io/V4MNrdQ the opened dropdown view
> p.s. this should be a quick fix

Done

File: source/widgets/language-switcher/LanguageSwitcher.module.scss [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    border-left: solid 1px var(--hint-text-color);
+
+    .languageDropdown {
+      border-left: 5.5px solid transparent;
+      border-right: 5.5px solid transparent;
+      border-top: 5.5px solid var(--primary-highlight-color);
+      height: 0;
+      left: 8px;
+      position: relative;
+      top: 19px;
```
Comment (a-rukin) [inline source/widgets/language-switcher/LanguageSwitcher.module.scss]: should be 18px

File: source/widgets/language-switcher/LanguageSwitcher.module.scss [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    }
+
+    .showLanguageDropdown {
+      border-left: 5.5px solid transparent;
+      border-right: 5.5px solid transparent;
+      border-bottom: 5.5px solid var(--primary-highlight-color);
+      height: 0;
+      left: 8px;
+      position: relative;
+      top: -7px;
```
Comment (a-rukin) [inline source/widgets/language-switcher/LanguageSwitcher.module.scss]: change to "-6"

File: source/widgets/language-switcher/LanguageSwitcher.module.scss [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+    top: 33px;
+    width: 92px;
+
+    li {
+      border-bottom: solid 1px var(--hint-text-color);
+      font-family: ProximaNova, sans-serif;
+      font-size: 12px;
+      font-weight: 300;
+      letter-spacing: 0.6px;
+      padding: 7.5px 10.3px;
```
Comment (a-rukin) [inline source/widgets/language-switcher/LanguageSwitcher.module.scss]: let's use "8px 10px 7px 10px" here

**Comment by a-rukin** [comment]
image
we don't need this separator here

File: source/widgets/language-switcher/LanguageSwitcher.module.scss (hunk 3 above)
Comment (DeeJayElly) [reply source/widgets/language-switcher/LanguageSwitcher.module.scss]: Done

File: source/widgets/language-switcher/LanguageSwitcher.module.scss (hunk 2 above)
Comment (DeeJayElly) [reply source/widgets/language-switcher/LanguageSwitcher.module.scss]: Done

File: source/widgets/language-switcher/LanguageSwitcher.module.scss (hunk 1 above)
Comment (DeeJayElly) [reply source/widgets/language-switcher/LanguageSwitcher.module.scss]: Done

**Comment by DeeJayElly** [comment]
> image
> 
> we don't need this separator here

Fixed

**Pull request merged**